catalogs/catalog-hadoop/src/main/java/org/apache/gravitino/catalog/hadoop/HadoopCatalogOperations.java

```diff
@@ -109,6 +118,34 @@ public String getKerberosRealm() {
     return kerberosRealm;
   }
 
+  public EntityStore getStore() {
+    return store;
+  }
+
+  public Map<NameIdentifier, UserInfo> getUserInfoMap() {
```

Comment:
Should we maintain these in the SecureHadoopCatalogOperations?

Reply:
let me see,  If we can move `initKerberos` to `SecureHadoopCatalogOperations`